### PR TITLE
Create elviscoly.json

### PR DESCRIPTION
**Pull request by elviscoly**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1051"></a>

### CODE REVIEW

**Comment by kumarsonsoff3** [comment]
Commit all these changes, and then you have done with this 🙃

File: public/data/elviscoly.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "YOUR NAME",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
```
Comment (syedsohan) [inline public/data/elviscoly.json:4]: Provide your bio.

File: public/data/elviscoly.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "YOUR NAME",
```
Comment (syedsohan) [inline public/data/elviscoly.json:2]:
```suggestion
  "name": "Iria Elvis",
```

File: public/data/elviscoly.json (hunk 1 above)
Comment (syedsohan) [inline public/data/elviscoly.json:4]: Provide your bio.

File: public/data/elviscoly.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/YOUR_GITHUB_USERNAME.png",
```
Comment (syedsohan) [inline public/data/elviscoly.json:5]:
```suggestion
  "avatar": "https://github.com/elviscoly.png",
```

File: public/data/elviscoly.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/YOUR_GITHUB_USERNAME",
```
Comment (syedsohan) [inline public/data/elviscoly.json:9]:
```suggestion
      "url": "https://github.com/elviscoly",
```

File: public/data/elviscoly.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/YOUR_TWITTER_USERNAME",
```
Comment (syedsohan) [inline public/data/elviscoly.json:14]: Provide your Twitter profile link if you have one!

**Review by syedsohan — CHANGES_REQUESTED** [review]
Provide Title, Date and Description of your Milestone too!

**Review by krshdev — CHANGES_REQUESTED** [review]
Looks like you haven't edited the template, can you update it with your info?

**Pull request closed without merging**